docs/upgrade_to_tails_2x.rst

```diff
@@ -0,0 +1,138 @@
+
+Insert your Transfer Device into a free USB port, and mount it by navigating to Places ▸ Computer, and clicking on the encrypted disk. You will be prompted to enter its passphrase.
+
+Open a Nautilus window with admin priviledges by navigating to Applications ▸ System Tools ▸ Root Terminal. At the terminal prompt, simply type `nautilus`.
+
+Type ``ctrl`` + ``l`` and navigate to `/live/persistence/TailsData_unlocked`.
+
+|Navigate to TailsData_unlocked|
+
+Open a new tab (``ctrl`` + ``t``) and navigate to your Transfer Device. Drag and drop the backup data from your Transfer Device onto the TailsData_unlocked tab.
+
+When copying a folder, select the *Apply this action to all files* option and click *Merge* to apply to all subfolders. Then you might have to select again the *Apply this action to all files* option and click *Replace* to apply to all files.
+
+In a root terminal, or as sudo, execute the following command to fix the ownership of your personal files:
+
+::
+
+	find /live/persistence/TailsData_unlocked/ -uid 1000 -exec chown -R 1000:1000 '{}' \;
+
+Reboot your primary Tails USB and verify *everything* is still there and accessible to you, including:
+
+-	KeePassX Database
+-	PGP keys
+
+.. note:: If you are restoring a Secure Viewing Station Tails USB, you may skip to Step 7. 
+	However, if you're restoring an Admin or Journalist Tails USB, continue to Step 6.
+
+6. Reinstall SecureDrop
+-----------------------
+
+Continue checking your persistent files for the following critical assets:
+
+-	Bookmarks in Tor Browser
+-	SecureDrop files, especially the ``torrc_additions`` file in ``~/Persistent/.securedrop``.
+-	If you're an admin, also be sure the files in ``~/Persistent/securedrop/install_files/`` are available.
+
+Shut down your Tails USB on the airgapped station and move to the computer you normally use to check for submissions. At this stage, all data has been migrated and it's safe to use this Tails USB on a networked computer.
+
+Boot up Tails once again with persistence and admin privileges.
+
+.. warning:: Copy ``~/Persistent/.securedrop/torrc_additions`` to a place like your desktop.
+	You'll need these old values for the following step.
+
+Re-install Securedrop with ``cd ~/Persistent/securedrop/tails_files && sudo ./install.sh``. Once completed, test your access to the Document Interface, and, if you're a Secure Drop admin, test your ssh connection to the application and monitor servers.
+
+If you did this correctly, move on to Step 7.
+
+7. Reformat the Transfer Device
+-------------------------------
+
+Repeat step 1 to restore a USB stick to a pristine state. While it probably isn't necessary to physically destroy a Transfer Device (because LUKS-encrypted data is very hard to forensically recover), you could smash the device with a hammer until the chips containing its flash memory are broken up into pieces before disposal.
```

Comment:
Also, I think your comment incorrectly copies the procedure from the wiki. The whole point of their procedure is to create an ephemeral volume with a random key (`-d /dev/urandom`) on top of the block device to be wiped. That way, `dd if=/dev/zero` *ultimately* writes a pseudorandom stream of bytes to the block device (since the zeros are being encrypted under a random, ephemeral key by dm-crypt).

Your steps don't provide a random key to `cryptsetup open`. Running that command prompts for the existing LUKS passphrase. As a result, the commands as you describe them *do not overwrite the LUKS header*. This is a problem because that is where the anti-forensic countermeasures are implemented (see section 5.2 of the [LUKS paper](http://tomb.dyne.org/New_methods_in_HD_encryption.pdf)).

In general, you should be ok as long as you can reliably overwrite the LUKS header. This saves you a lot of time, since the LUKS header is much smaller than the entire block device. This technique is documented in detail on the [Arch wiki](https://wiki.archlinux.org/index.php/Dm-crypt/Drive_preparation#Wipe_LUKS_header).

Unfortunately, ***reliably overwriting*** the LUKS header is not guaranteed to work on an SSD, because you can't reliably overwrite *anything* on an SSD. This is noted on the Arch Wiki, and they link to section 5.19 of the [cryptsetup FAQ](https://gitlab.com/cryptsetup/cryptsetup/wikis/FrequentlyAskedQuestions#5-security-aspects) for more info on securely wiping SSDs. Another good resource for background is [Wei '11](https://www.usenix.org/legacy/events/fast11/tech/full_papers/Wei.pdf).

This distinction is important because we're talking about wiping the transfer device, which is almost always a USB drive and therefore almost always using flash memory. Unfortunately, there are no easy answers here that I know of. I would probably stick with the technique from the cryptsetup wiki (also documented with some extra helpful detail on the [Arch wiki](https://wiki.archlinux.org/index.php/Dm-crypt/Drive_preparation#dm-crypt_wipe_on_an_empty_disk_or_partition)). According to Wei's paper,

>  In most cases, overwriting the entire disk twice was sufficient to sanitize the disk, regardless of the previous state of the drive.

That, combined with the anti-forensic properties of LUKS, seems "good enough" IMHO. Of course, the only way to be sure is to physically destroy the drive, which you've already mentioned as an option.